### PR TITLE
Add a 'required' attribute to paper-input.

### PR DESCRIPTION
**Pull request by jklein24**

### CODE REVIEW

File: paper-input.html [hunk 1]
```diff
@@ -90,6 +90,14 @@
       },
 
       /**
+       * Marks the `input` as required.
```
Comment (morethanreal) [inline paper-input.html]: How about "set to true to mark the `input` as required" to match the other doc string?

**Comment by morethanreal** [comment]
lgtm w/ nit

**Comment by morethanreal** [comment]
Please fix the nit and squash the commits.

**Comment by jklein24** [comment]
Done.

**Pull request merged**